src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -145,6 +161,38 @@ const ResearchProfile = () => {
           )}
         </>
       )}
+
+      <Divider />
+      <Typography id="project-label" variant="h2" sx={{ mt: '1rem' }}>
+        {t('my_page.my_profile.projects')} ({projectsQuery.data && projectsQuery.data.size})
```

Comment:
count nullstilles nå når man går til neste side, slik at innholdet hopper litt hit og dit. Dette blir en offentlig visning for alle så skulle gjerne oppført seg fint (selv om vi har mange svin på skogen fra før...) Mulig å få loading states her til å virke litt mer som det gjør for registreringer over?